api.rst

```diff
@@ -55,7 +68,25 @@ not           negates another operator, see below
 ============  =============================================
 
 
-To negate any operator, prefix it with :code:`not` like :code:`?a=not.eq.2`.
+To negate any operator, prefix it with :code:`not` like :code:`?a=not.eq.2` or :code:`?not.and=(a.gte.0,a.lte.100)` .
+
+For more complicated filters (such as those involving disjunctions) you will have to create a new view in the database, or use a stored procedure. For instance, here's a view to show "today's stories" including possibly older pinned stories:
```

Comment:
Also now that I look, this section exists in the docs as well at https://postgrest.com/en/v4.1/api.html#horizontal-filtering-rows - is it a rebase accident that it appears in the diff here?